docs/faststore/docs/sdk/analytics/send-analytics-event.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "sendAnalyticsEvent"
+---
+
+The `sendAnalyticsEvent` function triggers events in the browser, following the [Google Analytics 4 (GA4) data model](https://developers.google.com/analytics/devguides/collection/ga4/reference/events). This function is primarily used for ecommerce tracking but also supports custom events , serving as a centralized point for managing events
+
+{ /* Remember to add a link to 'custom events' once we have the following task done: EDU-13545 */ }
+
+> ⚠️ `sendAnalyticsEvent` doesn't send events to any analytics provider. To intercept events triggered by this function, use the [`useAnalyticsEvent`](https://developers.vtex.com/docs/guides/faststore/analytics-use-analytics-event) hook.
+
+## Import
+
+```tsx
+import { sendAnalyticsEvent } from '@faststore/sdk'
+```
+
+## Usage
+
+In the component related to the event, declare a callback function. In this function, define an event object with the desired event type (example: `add_to_cart`) and call `sendAnalyticsEvent`. Then, pass the related event as an argument. Finally, call the callback function in the event trigger (example: `onClick`).
+
+Consider the following example of an `add_to_cart` event triggered by the `button` component:
+
+```tsx
+import { useCallback } from 'react'
+import { sendAnalyticsEvent } from '@faststore/sdk'
+
+const MyComponent = () => {
+  const addToCartCallback = useCallback(() => {
+    /* ... */
+
+    const addToCartEvent = {
+      type: 'add_to_cart',
+      data: {
+        items: [
+          /* ... */
+        ],
+      },
+    }
+
+    sendAnalyticsEvent(addToCartEvent)
+  }, [])
```

Comment:
suggestion: I believe it would be better to put the dynamic import on this doc [[ref]](https://github.com/vtex/faststore/pull/2472)
cc @eduardoformiga what do you think? 
```suggestion
  const addToCartCallback = useCallback(() => {
   import('@faststore/sdk').then(({ sendAnalyticsEvent }) => {
    /* ... */

    const addToCartEvent = {
      type: 'add_to_cart',
      data: {
        items: [
          /* ... */
        ],
      },
    }

    sendAnalyticsEvent(addToCartEvent)
    })
  }, [])
```